### DISCUSSION TITLE
How can i run LocalAI on Windows?

### DISCUSSION THREAD

**Discussion by shamio** [mudler/LocalAI#602] [Q&A]
I couldn't find any Windows  version. Could someone help me to install it on Windows? and if its not ready for Windows yet, could the developers provide a Windows version too?

**Comment by mike-niemand** [comment]
Run it in a docker container.

**Reply by BradKML** [reply]
TBH seconding this since Docker Desktop is very easy to use

**Comment by lunamidori5** [comment]
@shamio If you already have docker desktop installed check out the Midori AI subsystem, it can install LocalAI with all the needed stuff for you with little to no command line! - https://io.midori-ai.xyz/subsystem/manager/

**Reply by BradKML** [reply]
Nice self-promo buddy, also can you cover Xorbits interface (for like rarankers) and A1111 variants (for stable diffusion)?

**Reply by lunamidori5** [reply]
@BradKML  Sure, give me a bit with our new servers and rework! (we already have invokeai but any111 is on the list!)

**Reply by BradKML** [reply]
Beat Big-AGI and especically Pinokio for me (also StabilityMatrix if you are artistic) cus ain't nobody got time to setup backend for things like Dify, Autogen, MetaGPT, AgentGPT, XAgent, BabyAGI, or SuperAGI (yeah long list)

**Reply by lunamidori5** [reply]
@BradKML autogpt added yesterday (doing server upgrades so working on that) but more are planed
